Mac/Inspector/WebFeedInspectorViewController.swift

```diff
@@ -113,6 +113,7 @@ extension WebFeedInspectorViewController: NSTextFieldDelegate {
 			return
 		}
 		feed.editedName = nameTextField.stringValue
+		windowTitle = feed.editedName ?? NSLocalizedString("Feed Inspector", comment: "Feed Inspector window title")
```

Comment:
@brentsimmons I'm not 100% of this WebFeedInspectorViewController change. I'm tapping into the textfield delegate's controlTextDidChange method to make sure the window title updates as the feed's name is changed. If this seems like a reasonable approach, then I think this PR is ready for re-review!